### PR TITLE
tips and tricks added about random theme using Z-shell

### PR DESCRIPTION
**Pull request by krishnadevz**
## Summary of the Pull Request
<p>Simply i added trick for random theme for Ubuntu distro in WSL terminal by using Z-shell and other things i provided in code .it changes theme every single time when you open another terminal window that's all.</p> 
## References

* [Z-shell](https://github.com/ohmyzsh/ohmyzsh)

## PR Checklist
* [ ] Closes #xxx
* [x] CLA signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/Terminal) and sign the CLA
* [ ] Tests added/passed
* [ ] Requires documentation to be updated
* [ ] I've discussed this with core contributors already. If not checked, I'm ready to accept this work might be rejected in favor of a different grand plan. Issue number where discussion took place: #xxx

### CODE REVIEW

**Comment by skyline75489** [comment]
I like the idea. But this does not seem to really fit in with Windows Terminal trick. It's more about zsh itself. Don't get me wrong. I love zsh and use it almost everyday. But you can zshell anywhere you want, right? Be it iTerm2, Gnome Terminal. Konsole, etc. You know what I mean.

**Comment by krishnadevz** [comment]
> I like the idea. But this does not seem to really fit in with Windows Terminal trick. It's more about zsh itself. Don't get me wrong. I love zsh and use it almost everyday. But you can zshell anywhere you want, right? Be it iTerm2, Gnome Terminal. Konsole, etc. You know what I mean.

Yup I understand but in readme Microsoft terminal member said add tips and tricks and this trick can help others that's why added what do you think ?

File: doc/user-docs/index.md [hunk 1]
```diff
@@ -87,4 +87,12 @@ For an introduction to the various settings, see [Using Json Settings](UsingJson
 2. Terminal zoom can be changed by holding <kbd>Ctrl</kbd> and scrolling with mouse.
 3. Background opacity can be changed by holding <kbd>Ctrl</kbd>+<kbd>Shift</kbd> and scrolling with mouse. Note that acrylic transparency is limited by the OS only to focused windows.
 4. Open Windows Terminal in current directory by typing `wt -d .` in the address bar.
-5. Please add more Tips and Tricks.
+5. Random theme in terminal by using Zsh(Z-shell) for Ubuntu distro [LearnMoreAboutZ-shell](https://github.com/ohmyzsh/ohmyzsh)
```
Comment (WSLUser) [inline doc/user-docs/index.md:90]: Remove `for Ubuntu distro`. Zsh can be used everywhere.

File: doc/user-docs/index.md [hunk 2]
```diff
@@ -87,4 +87,12 @@ For an introduction to the various settings, see [Using Json Settings](UsingJson
...
+* `sudo apt install zsh` 
+* `sh -c "$(curl -fsSL https://raw.githubusercontent.com/ohmyzsh/ohmyzsh/master/tools/install.sh)"`
+* `sudo apt-get install fonts-powerline`
+* `code ~/.zshrc` # to customize it
+* Then change the `ZSH_THEME="random"`. Your terminal sessions always having unique theme.
+6. Please add more tips & tricks
```
Comment (WSLUser) [inline doc/user-docs/index.md:96]: Forgot the period at the end.

Comment (krishnadevz) [reply doc/user-docs/index.md:96]: Sorry but What do you mean by period I didn't get it ?

Comment (WSLUser) [reply doc/user-docs/index.md:96]: > Please add more tips & tricks

Please add more tips & tricks.

Comment (krishnadevz) [reply doc/user-docs/index.md:96]: I think you didn't seen last six number point I already mentioned there.about please add more tips and tricks. Thank you 😊

File: doc/user-docs/index.md [hunk 3]
```diff
@@ -87,4 +87,12 @@ For an introduction to the various settings, see [Using Json Settings](UsingJson
 2. Terminal zoom can be changed by holding <kbd>Ctrl</kbd> and scrolling with mouse.
 3. Background opacity can be changed by holding <kbd>Ctrl</kbd>+<kbd>Shift</kbd> and scrolling with mouse. Note that acrylic transparency is limited by the OS only to focused windows.
 4. Open Windows Terminal in current directory by typing `wt -d .` in the address bar.
-5. Please add more Tips and Tricks.
+5. Random theme in terminal by using Zsh(Z-shell) [LearnMoreAboutZ-shell](https://github.com/ohmyzsh/ohmyzsh)
+* `sudo apt install zsh` 
+* `sh -c "$(curl -fsSL https://raw.githubusercontent.com/ohmyzsh/ohmyzsh/master/tools/install.sh)"`
+* `sudo apt-get install fonts-powerline`
+* `code ~/.zshrc` # to customize it
```
Comment (WSLUser) [inline doc/user-docs/index.md:94]: Also change the word `code` to vim or nano. (I personally prefer vim.)

Comment (krishnadevz) [reply doc/user-docs/index.md]: I changed to vim but I use mostly vscode for my Web development thing and I  ❤ vscode .that's all about work and perspective😊

**Comment by zadjii-msft** [comment]
Hey @krishnadevz

Thanks for the contribution! However, I think I tend to agree with what @skyline75489 said. I think we intended that doc to contain tips & tricks _specific to the Windows Terminal_, not just general commandline tips and tricks. This is certainly a neat little tip, but I'm not sure it's really at home here. 

Thanks!

**Pull request closed without merging**